.pipelines/.vsts-vhd-automation.yaml

```diff
@@ -5,61 +5,146 @@ pool:
+        env:
+          MAPPED_ADO_PAT: $(PAT-aksdevassistant)
+        displayName: 'az devops login'
+      - bash: |
+          echo "PR for Image Bumping, Official Branch Cutting"
+          /bin/bash vhdbuilder/scripts/automate_version_bump.sh $(IMAGE_VERSION) $MAPPED_GITHUB_PAT "${BUILD_ID[@]}"
+        env:
+          MAPPED_GITHUB_PAT: $(GITHUB_PAT)
+        displayName: 'bump image version and create official branch'
+  - stage: generate_release_notes
+    dependsOn: []
+    condition: eq('${{ parameters.ReleaseNotes }}', true)
+    jobs:
+    - job: automation
+      timeoutInMinutes: 30
+      steps:
+      - bash: |
+          set -x
+          echo "Removing existing go environment"
+          sudo rm -r /usr/local/go
+          GOLANG_VERSION="1.22.2"
+          echo "Downloading ${GOLANG_VERSION}"
+          curl -O "https://dl.google.com/go/${GOLANG_VERSION}.linux-amd64.tar.gz"
 
-        echo "unpacking go"
-        sudo mkdir -p /usr/local/go
-        sudo chown -R "$(whoami):$(whoami)" /usr/local/go 
-        sudo tar -xvf "${GOLANG_VERSION}.linux-amd64.tar.gz" -C /usr/local
-        rm "${GOLANG_VERSION}.linux-amd64.tar.gz"
-  displayName: 'Clean up go environment'
-- bash: |
-        az extension add -n azure-devops
-        echo $MAPPED_ADO_PAT | az devops login --organization=https://dev.azure.com/msazure
-        az devops configure --defaults organization=https://dev.azure.com/msazure project=CloudNativeCompute
-  env:
-        MAPPED_ADO_PAT: $(PAT-aksdevassistant)
-  displayName: 'az devops login'
-- bash: |
-        echo "PR for Image Bumping, Official Branch Cutting"
-        /bin/bash vhdbuilder/scripts/automate_version_bump.sh $(IMAGE_VERSION) $MAPPED_GITHUB_PAT "${BUILD_ID[@]}"
-  env:
-        MAPPED_GITHUB_PAT: $(GITHUB_PAT)
-  displayName: 'Image Version Bumping'
-  condition: eq('${{ parameters.ImageBump }}', true)
-- bash: |
-        echo "PR for Release Notes"
-        /bin/bash vhdbuilder/scripts/automate_release_notes.sh $(IMAGE_VERSION) "${BUILD_ID[@]}" $MAPPED_GITHUB_PAT
-  env:
-        MAPPED_GITHUB_PAT: $(GITHUB_PAT)
-        SKIP_LATEST: $(SKIP_LATEST_RELEASE_NOTES_UPDATE)
-  displayName: 'Release Notes'
-  condition: eq('${{ parameters.ReleaseNotes }}', true)
-- bash: |
-        echo "Triggering ev2 artifacts pipeline"
-        /bin/bash vhdbuilder/scripts/automate_ev2pipeline_trigger.sh "${BUILD_ID[@]}"
-  displayName: 'ev2 Artifact Pipeline Trigger'
-  condition: eq('${{ parameters.ArtifactTrigger }}', true)
+          echo "unpacking go"
+          sudo mkdir -p /usr/local/go
+          sudo chown -R "$(whoami):$(whoami)" /usr/local/go 
+          sudo tar -xvf "${GOLANG_VERSION}.linux-amd64.tar.gz" -C /usr/local
+          rm "${GOLANG_VERSION}.linux-amd64.tar.gz"
+        displayName: 'clean up go environment'
+      - bash: |
+          az extension add -n azure-devops
+          echo $MAPPED_ADO_PAT | az devops login --organization=https://dev.azure.com/msazure --verbose
```

Comment:
does the `--verbose` flag print the PAT ?

Also, I see that the PAT is create in a different bash/stage, if we have access to the ENV var, how come we are loosing the login to azuredevops performed above ?

Reply:
the login in this step propagates to the following steps within the same job

Reply:
and no, `--verbose` does not print any secrets